Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
```

Comment:
I regard this is as a human-parseable diagnostic tool and having libSystem in the output interferes with that goal.

I think a list of mach-o binaries could be implemented as `brew ls --mach-o` but I don't think that's what this should be.